src/tests/functional/inference_engine/ir_serialization/partial_shape_deserialization.cpp

```diff
@@ -348,7 +350,8 @@ TEST_F(PartialShapeDeserialization, ShapeWithBoundariesTestWrongDim) {
     </edges>
 </net>
 )V0G0N";
-    ASSERT_THROW(getWithIRFrontend(model), InferenceEngine::Exception);
+    // TODO: change to ov::Exception (69781)
+    ASSERT_ANY_THROW(getWithIRFrontend(model));
```

Comment:
why not `ov::Exception`?
maybe better to completely remove `IE_THROW` from IR FE?

Reply:
Yes, there is a ticket for this to remove 'inference-engine' dependency on IR frontend (69781). I believe it will be done under this